placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -439,4 +443,23 @@ public void setLanguageCode(@Nullable String languageCode) {
         this.languageCode = languageCode;
         api.setLanguageCode(this.languageCode);
     }
+
```

Comment:
Won't let me comment on a line not in the review (thanks github) but can we also make this class extend from `AppCompatAutocompleteTextView`?